template.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
```

Comment:
this doesn't get built on its own?

Reply:
It could but then it has a bunch of stuff in it that is unnecessary with only limited customization options. 
There is also no good option to inline JS when it builds the html itself. 
Like this, customization is much easier for ppl and I have a placeholder for the inlined JS :)